### ISSUE TITLE
The models to initialize the segmentation network

### ISSUE THREAD

**Issue by Zhengyangnuaa** [PengtaoJiang/OAA#9]
Would you tell me how did you get the models mentioned in 3.Models, about initializing the VGG-based DeepLab-LargeFOV network.
and initializing the ResNet-based DeepLab-LargeFOV.
Do they come from the same source as http://liangchiehchen.com/projects/DeepLab_Models.html?

**Comment by PengtaoJiang** [comment]
Hello, they all come from the deeplab webpage.

**Comment by Zhengyang1995** [comment]
So are they pretrained in a fully supervised way?

**Comment by Zhengyang1995** [comment]
so I think it is pretrained in a classification task in MS-COCO?

**Comment by PengtaoJiang** [comment]
@Zhengyang1995  Yes, the resnet is pretrained for classification on MS-COCO.  The vgg is pretrained for classification on ImageNet (We don't use the pretrained model on COCO for vgg).
